cmd/vsphere-cloud-controller-manager/main.go

```diff
@@ -165,21 +133,12 @@ func main() {
 	// TODO: once we switch everything over to Cobra commands, we can go back to calling
 	// utilflag.InitFlags() (by removing its pflag.Parse() call). For now, we have to set the
 	// normalize func and add the go flag set by hand.
-	pflag.CommandLine.SetNormalizeFunc(flag.WordSepNormalizeFunc)
+	pflag.CommandLine.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 	// utilflag.InitFlags()
 	logs.InitLogs()
 	defer logs.FlushLogs()
 
-	// Set cloud-provider flag to vsphere
```

Comment:
looks like that was just leftover code
I tried changing the flag value to "vsphere" or "vsphere-paravirtual", there is no difference in the log/behavior and didn't see any error log during testing, maybe we could remove this?

Reply:
Makes sense to me

Reply:
I think this is to cover the case when `--cloud-provider` is not set, and have it then default to `vsphere`. Have you tried that?

Reply:
ah you are right, I added it back
if --cloud-provider is not set, we will get errors